README.md

```diff
@@ -1,49 +1,16 @@
-* **Push your solution to the newly created branch.**
- * See resources at the bottom of the page for more info on how to do this.
-* **Create a pull request** to allow someone to pull your solution onto the repo.
- * Click on the green pull request button which is next to the new branch button. Set the "base" to "master" and the "compare" to your new branch name "new-member-x". 
-* **Create a new simple JS problem** titled "new-member-x+1" for the next new member to practice with.
- * Post your simple problem as a new issue:  set the title as "new-member-x+1" and include instructions in the description.
- * Remember, keep this simple: for new members this is more about getting comfortable with github.
- 
-**If there is no issue waiting for you titled "new-member-x" then create an issue titled "new member in need of problem"**
-
-
-### Here is the general workflow for this repository:
-
- - For each new challenge, the author creates a new branch named after the task and creates an issue describing the challenge.
- - If any of the contributors want to take the challenge, a contributor replies to the issue so others won't try to push their solutions at the same time - this 'reserves' them a spot for solving the problem.
- - When a solution is ready, the changes can be pushed to the branch named after the task and then a pull request created (from that branch to master). So even if the solution isn't correct, it won't be pushed to master branch yet and can be corrected easily.
- - If the author of the challenge decides that the solution is correct, he/she can merge the pull request into the master.
- 
-As per your preference you can create a working branch of master within this repository or "fork" this repository into your own GitHub environment. Please make sure to synchronize a fork with upstream repository before creating a pull request so there will be no any conflicts in request. See the guide below for assistance.
-
-#Some Resources:
-**Here's a good read on Git:**
-
-https://github.com/NataliaLKB/learn-git-basics
-
-**A nice tutorial on resolving conflicts when merging
-
-http://www.gitguys.com/topics/merging-with-a-conflict-conflicts-and-resolutions/ 
-
-**Youtube introduction to GitHub (30 minutes)**
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/codingforeveryone/js?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-https://www.youtube.com/playlist?list=PLfdtiltiRHWFEbt9V04NrbmksLV4Pdf3j
+# Using this repository
 
-**GitHub help on 'Fork a repo'**
-https://help.github.com/articles/fork-a-repo/
+This repository is for becoming familiar with GitHub by completing JavaScript problems.
+There is a process to follow for contributing. The basics are outlined below, and full details/help can be found in [the CONTRIBUTING.md file](CONTRIBUTING.md).
 
```

Comment:
This all looks great, maybe add something here like "Join our chat room on gitter, we'd love to hear from you!", just to make it very clear this is a friendly and sociable repo